.github/workflows/ci.yml

```diff
@@ -86,9 +105,9 @@ jobs:
         with:
           fail_ci_if_error: false
 
-
   try-runtime:
     runs-on: ubuntu-latest-16-cores
+    timeout-minutes: 45
```

Comment:
What happens when it times out? Is it a failure?

Reply:
Yes it will show as ❌ in the summary, but when one open it will be a warning notification
![image](https://user-images.githubusercontent.com/3679593/208001999-647f1f1d-ed45-413e-b313-08125aaf2d5d.png)

Since we push several times when developing this way it is easy to see if it was just once it got stuck in network traffic or it is a failing trend.